ui/tests/unit/models/transit-key-test.js

```diff
@@ -19,7 +19,7 @@ module('Unit | Model | transit key', function(hooks) {
       })
     );
 
-    let supportedActions = model.get('supportedActions');
+    let supportedActions = model.get('supportedActions').map(k => k.name);
```

Comment:
It might also be worth testing for the transit-card-container and transit-card classes with icons in them.

Reply:
ah, nice suggestion! i opted not to do that here since this is a unit test that is testing the transit model itself, and not DOM elements. testing whether DOM elements exist happens inside acceptance or component tests, which in this case is here: https://github.com/hashicorp/vault/blob/d4362955cd60c9db1029aaedc25ba8cafcbe4ef5/ui/tests/acceptance/transit-test.js#L265

however, i have updated the above test to check for the card title. once i get the structure icons updated i'll look into checking for an icon as well. :D

Reply:
ah, nice suggestion! i opted not to do that here since this is a unit test that is testing the transit model itself, and not DOM elements. testing whether DOM elements exist happens inside acceptance or component tests, which in this case is here: https://github.com/hashicorp/vault/blob/d4362955cd60c9db1029aaedc25ba8cafcbe4ef5/ui/tests/acceptance/transit-test.js#L265

however, i have updated the above test to check for the card title. once i get the structure icons updated i'll look into checking for an icon as well. :D